docs/cloud/satellites/managing.md

```diff
@@ -115,6 +115,26 @@ earthly sat launch <satellite-name>
 
 The newly launched satellite will always get the latest version available.
 
+### Managing a satellite's state
+
+To save costs, satellites automatically enter a "sleep" state after 30 min of inactivity. While a satellite is asleep, you are not billed for any compute minutes.
+
+The satellite will automatically "wake up" when a new build is started while it's in a sleep state. This is visible during the `Init` phase of the Earthly log.
+
+If you want more fine-grain control over your Satellite's state, you can also manually put it to sleep using the command:
+
+```bash
+earthly sat sleep <satellite-name>
+```
+
+Similarly, a Satellite can be manually woken up using:
+
+```bash
+earthly sat wake <satellite-name>
+```
+
+Note that the [`inspect`](#Checking status of a satellite) command will show you if a Satellite is currently awake or asleep.
```

Comment:
I don't think this link works - you might need to check the preview to see what the correct anchor should be here.

Reply:
I think I have the syntax right now, but it doesn't seem like it's creating the new preview link?

Reply:
Not sure why - maybe it needs yet another commit?

Reply:
It looks right now btw, triggered it with some dummy commits